workflows/data_pipelines/elasticsearch/data_enrichment.py

```diff
@@ -1,42 +1,39 @@
 import json
 import logging
+
 from slugify import slugify
 
-from dag_datalake_sirene.workflows.data_pipelines.elasticsearch.clean_data import (
-    drop_duplicates_dirigeants_pm,
-    drop_duplicates_personnes_physiques,
-    unique_qualites,
-)
-from dag_datalake_sirene.workflows.data_pipelines.elasticsearch.es_fields import (
-    get_elasticsearch_field_name,
+from dag_datalake_sirene.helpers.geolocalisation import (
+    transform_coordinates,
 )
 from dag_datalake_sirene.helpers.utils import (
     drop_exact_duplicates,
     get_empty_string_if_none,
+    load_file,
+    sqlite_str_to_bool,
     str_to_bool,
     str_to_list,
-    sqlite_str_to_bool,
 )
-from dag_datalake_sirene.helpers.geolocalisation import (
-    transform_coordinates,
+from dag_datalake_sirene.workflows.data_pipelines.elasticsearch.clean_data import (
+    drop_duplicates_dirigeants_pm,
+    drop_duplicates_personnes_physiques,
+    unique_qualites,
+)
+from dag_datalake_sirene.workflows.data_pipelines.elasticsearch.es_fields import (
+    get_elasticsearch_field_name,
 )
-
-labels_file_path = "dags/dag_datalake_sirene/helpers/labels/"
-
-
-def load_file(file_name: str):
-    with open(f"{labels_file_path}{file_name}") as json_file:
-        file_decoded = json.load(json_file)
-    return file_decoded
-
 
 sections_NAF = load_file("sections_codes_naf.json")
 mapping_dep_to_reg = load_file("dep_to_reg.json")
 mapping_role_dirigeants = load_file("roles_dirigeants.json")
 mapping_commune_to_epci = load_file("epci.json")
-nature_juridique_service_public = set(load_file("nature_juridique_service_public.json"))
-service_public_whitelist = set(load_file("service_public_whitelist.json"))
-service_public_blacklist = set(load_file("service_public_blacklist.json"))
+nature_juridique_service_public = set(load_file("administration_nature_juridique.json"))
+service_public_whitelist = set(load_file("administration_siren_whitelist.json"))
+service_public_blacklist = set(load_file("administration_siren_blacklist.json"))
+L100_3_siren_whitelist = set(load_file("administration_L100-3_siren_whitelist.json"))
+excluded_nature_juridique_L100_3 = set(
+    load_file("administration_L100-3_nature_juridique_exclue.json")
+)
```

Comment:
nit: more consistant naming

```suggestion
administration_nature_juridique_whitelist = set(load_file("administration_nature_juridique.json"))
administration_siren_whitelist = set(load_file("administration_siren_whitelist.json"))
administration_siren_exclusion = set(load_file("administration_siren_blacklist.json"))
l100-3_siren_whitelist = set(load_file("administration_L100-3_siren_whitelist.json"))
l100-3_nature_juridique_exclusion = set(
    load_file("administration_L100-3_nature_juridique_exclue.json")
)
```

Reply:
We can't have a hyphen in var names either :/ I will merge now because I tested this and will add in to do better naming. I would like to ship this asap 😅